shoop/admin/static_src/picotable/picotable.js

```diff
@@ -551,6 +551,16 @@ const Picotable = (function(m, storage) {
         ]);
     }
 
+    function parseGetParamsString(paramsString) {
+        var params = {};
```

Comment:
And actually, you can just use `m.route.parseQueryString` (from Mithril) instead of rolling your own, here. :)